Assets/VRTK/Scripts/Helper/VRTK_Hips.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class VRTK_Hips : MonoBehaviour {
+
+	public Transform playerHead;
+	public float HeadOffset = -0.35f;
+	[Header("Optional")]
+	public Transform ReferenceUp;
+
+	void Update()
+	{
+		if(playerHead == null)
+		{
+			return;
```

Comment:
Consider setting the head in Start/Awake instead of forcing the user to set it themselves. If the user has explicitly set it then obviously you wouldn't want to override it in that case. This would still allow the customization, but removes one bug prone step if the use forgets to manually set it themselves.